guides/release/reference/accessibility-guide.md

```diff
@@ -0,0 +1,109 @@
+The purpose of this guide is to give every Ember developer a starting point for making your Ember application minimally accessible. As accessibility can be a complex issue, it should be noted that this guide only covers a some of the basics, and should not be considered a comprehensive guide. A good place to start is [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) provided by the Web Accessibility Initiative (WAI).
+
+## Ember Application Configuration
+
+### Optional Feature: Application Template Wrapper
+
+If you are using the [application template wrapper](#toc_optional-features) enabled (default state), then you will need to add certain aria roles to your [landmark regions](https://www.w3.org/WAI/PF/aria/roles#landmark_roles), even if you are using native HTML elements, because those regions are not the direct child descendant of the body element (they are the children of the div that wraps the Ember app).
+
+If you disable the [application template wrapper](#toc_optional-features), you will not need to add role attributes to your landmark regions when they are the direct descendant of the body element, and they are using native HTML elements. This is the preferred approach for accessible applications. 
+
+**Application Template Wrapper Disabled** _(preferred)_
+```html
+<body>
+  <header></header>
+  <main></main>
+  <footer></footer>
+</body>
+```
+
+**Application Template Wrapper Enabled**
+```html
+<body>
+  <div class="ember-view">
+    <header role="banner"></header>
+    <main role="main"></main>
+    <footer role="contentinfo"></footer>
+  </div>
+</body>
+```
+
+> Note: To learn more about landmark roles and how to use them: [https://www.w3.org/WAI/PF/aria/roles#landmark_roles](https://www.w3.org/WAI/PF/aria/roles#landmark_roles). Still need more help? Visit the #topic-a11y channel in [Ember chat](https://emberjs.com/community/). 
+
+### Ember Application vs role="application"
+
+An important thing to note in this section is this: "application" in Ember development and "application" in landmark roles have two _very_ different meanings. 
+
+The TL;DR? Don't use `role="application"` until you have done your research and know exactly how it is to be used correctly (if at all). There are very few use cases where the role of application is appropriate. 
```

Comment:
Do we have a policy about using potentially ambiguous abbreviations? I actually didn't know what TLDR meant until embarrassingly recently 🙈